openassessment/assessment/test/test_peer.py

```diff
@@ -152,13 +152,14 @@
     }
 }
 
+
 @ddt
 class TestPeerApi(CacheResetTest):
     """
     Tests for the peer assessment API functions.
     """
 
-    CREATE_ASSESSMENT_NUM_QUERIES = 58
+    CREATE_ASSESSMENT_NUM_QUERIES = 53
```

Comment:
Yep -- we switched from using a custom "nested serializer" to having a single `create()` method on the parent rubric.  The code in the `create()` method needed to make fewer queries to get the same job done, because it could bulk-create the nested models.